### PR TITLE
[WIP] alternate to removing xenobio magicarps: replaces xenobio magicarps with gremlin locker creatures that fuck with you

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

alternate to #49577 

xenobio magicarps are replaced by shitty little gremlins with locker masks that shoot locker bolts and when hit have a chance to pop into a locker on it's tile. there's a 50% chance the locker contains him, and a 50% chance he teleported away and the locker contains a grenade

as they are a netherlink creature, they are spawned from the netherlink portal

## Why It's Good For The Game

i was sad there were no alternates, and we didn't use any of the cool staff effects we had in store

## Changelog
:cl:
add: shitty gremlin locker dude to the xenobio pool, they're also found in the netherlink spawner
del: xenobio magicarps
/:cl:

### CODE REVIEW

**Comment by IndieanaJones** [comment]
Very based, turning off my PR

File: code/modules/mob/living/simple_animal/hostile/netherworld.dm [hunk 1]
```diff
@@ -165,7 +209,7 @@
 	max_mobs = 15
 	icon = 'icons/mob/nest.dmi'
 	spawn_text = "crawls through"
-	mob_types = list(/mob/living/simple_animal/hostile/netherworld/migo, /mob/living/simple_animal/hostile/netherworld, /mob/living/simple_animal/hostile/netherworld/blankbody)
+	mob_types = list(/mob/living/simple_animal/hostile/netherworld/migo, /mob/living/simple_animal/hostile/netherworld, /mob/living/simple_animal/hostile/netherworld/blankbody, /mob/living/simple_animal/hostile/netherworld/locker)
```
Comment (tralezab) [inline code/modules/mob/living/simple_animal/hostile/netherworld.dm]: this is a change i should really mention one sec

**Comment by CRITAWAKETS** [comment]
0/10, isnt a gremlins reference

**Comment by carshalash** [comment]
Port /vg/ gremlins.

**Comment by cacogen** [comment]
sounds like some gayass shit to me

**Comment by OverlyGeneric** [comment]
ah yes, the spirit of a bullied kill from hell.

revenge after all those years being shoved in lockers.

**Comment by OverlyGeneric** [comment]
ah yes, bullied powergamer nerd who was shoved in lockers. he has come to take his petty revenge.

A very good addition.

**Comment by TheVekter** [comment]
By "locker bolts" are we talking about the stuff the Staff of Locker fires? If so, 10/10. Prime chicanery. I love it.

**Comment by WarlockD** [comment]
Meh, I am calling it first, someone will remove this from xeno in 3 months because they were killed by it.

**Comment by Tharcoonvagh** [comment]
Closed?

**Comment by interestingusernam3** [comment]
Sad!

**Pull request closed without merging**